index.html

```diff
@@ -4,42 +4,40 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/styles.css" />
     <title>Document</title>
```

Comment:
you could give your app a title
```suggestion
    <title>Quiz App</title>
```